ndarray/src/main/java/org/tensorflow/ndarray/impl/sparse/AbstractSparseNdArray.java

```diff
@@ -402,6 +402,26 @@ public boolean equals(Object obj) {
     return values.equals(other.values);
   }
 
+  /**
+   * A String showing the type, default value, number of elements and
+   * the dense shape of this sparse ndarray.
+   * @return A string containing the type, default value, number of elements and shape.
+   */
+  @Override
+  public String toString() {
+    long numElements = values == null ? 0 : values.size();
+    String strDefault;
+    if (defaultValue == null) {
+      strDefault = "<null>";
+    } else if (defaultValue instanceof Number){
```

Comment:
Nit: We don't have spotless installed yet in NdArray but I have a feeling that the missing space before the opening brace will generate an error if we do

Reply:
Done.